### ISSUE TITLE
import map from minecraft-generated and locally stored

### ISSUE THREAD

**Issue by rkiddy** [PaperMC/Paper#1678]
I have a world that I have been running in "local mode" and I want to copy the world data up into a paperclip. I can see that things match up between the local data (files and directories) and the server data. But they have odd differences also.

For example, on a local, you have:
     DIM-1
     DIM-1/region
     DIM-1/data
On paperclip, I seem to have:
     world/DIM-1/data
     world_nether/DIM-1/region

Is it as simple as that? But I also see "world_nether/DIM-1/data". So there are two of them? But the paperclip world is older and the directories may be only created as needed.

I cannot tell if the structures match up, or if there is going to be some indeterminateness to deal with.

The last time I tried to move a large world, the nether got reset. Would like to be able to avoid that.

Any ideas?

**Comment by electronicboy** [comment]
Copy the save folder itself over and either rename it to world or update the level-name in server.properties and the server will move the dimensions out to their correct places on first boot

**Comment by rkiddy** [comment]
Well, not quite. I get:

[13:03:58 WARN]: Unable to migrate world data.

And, as I expected, the world was not restored. O well.

**Comment by electronicboy** [comment]
are you sure that you don't already have _the_end and _nether worlds generated?
That error means that it failed to copy to copy the folders into place

**Comment by ShaneBeee** [comment]
For poops and hahas I did a test to see how this would go, since Ive never done it before.
I went on single player created a world, went into the nether and built a beautiful structure (JK it was 3 oak blocks)

Then set up a new paper server. Start Stop.... deleted the nether and end worlds. 
Deleted the contents of the world folder
Open up my "single player" saved world. copy the entire contents and pasted it into my "world" folder on my paper server. 

Started up my server

I get this
```
[13:20:10 INFO]: ---- Migration of old nether folder required ----
[13:20:10 INFO]: Unfortunately due to the way that Minecraft implemented multiworld support in 1.6, Bukkit requires that you move your nether folder to a new location in order to operate correctly.
[13:20:10 INFO]: We will move this folder for you, but it will mean that you need to move it back should you wish to stop using Bukkit in the future.
[13:20:10 INFO]: Attempting to move world/DIM-1 to world_nether/DIM-1...
[13:20:10 INFO]: Success! To restore nether in the future, simply move world_nether/DIM-1 to world/DIM-1
[13:20:10 INFO]: ---- Migration of old nether folder complete ----
[13:20:10 INFO]: -------- World Settings For [world_nether] --------
```
Logged into my server, went to the nether and my beautiful build is there.
BUENO

Now since I have tested this and know it works, I have a question.
Your single player game, was it 1.13.2 and then migrated to a 1.13.2 paper server?

If not, was it by any chance a 1.12.2 or lower single player game migrated to 1.13.2 paper?
If yes, I believe (dont quote me on this) but Spigot stated you would need to convert your single player 1.12.2 world to 1.13.2 via vanilla first (ie in single player open it up in a single player 1.13.2 game) this will convert the world to 1.13.2 then you can migrate it to paper.
IF no, then ignore this last part!
